apps/web/src/services/user/setupService.test.ts

```diff
@@ -74,15 +73,6 @@ describe('setupService', () => {
       where: eq(apiKeys.workspaceId, workspace.id),
     })
     expect(createdApiKey).toBeDefined()
-
-    // Check provider API key creation
-    const createdProviderApiKey =
-      await database.query.providerApiKeys.findFirst({
-        // @ts-expect-error - drizzle-orm types are not up to date
-        where: eq(providerApiKeys.workspaceId, workspace.id),
-      })
-    expect(createdProviderApiKey).toBeDefined()
-    expect(createdProviderApiKey?.authorId).toBe(user.id)
```

Comment:
Not happening now?

Reply:
it's optional yes, it won't happen in open source